### ISSUE TITLE
Storybook babel-loader error

### ISSUE THREAD

**Issue by bartduisters** [nrwl/nx#3164]
## Current Behavior

I follow the guide to add Storybook with NX and get an error when trying to run it.
![image](https://user-images.githubusercontent.com/43420049/84483338-82a44980-ac99-11ea-947d-686d9ce28806.png)

## Expected Behavior
I follow the guide to add Storybook with NX and expect to be able to run it without an error.

## Steps to Reproduce
- In a nx repo add storybook ```yarn add --dev @nrwl/storybook```
- Generate a Storybook configuration for a specific project ```npx nx g @nrwl/angular:storybook-configuration project-name```
- Run Story book for that given project ```npx nx run project-name:storybook```

### Environment
![image](https://user-images.githubusercontent.com/43420049/84483438-9ea7eb00-ac99-11ea-9cc4-dfdae9a2b40f.png)

### Workaround
Add 'babel-loader' to devDependencies. I guess this should be installed as dependency automatically.

**Comment by mandarini** [comment]
Hi there @bartduisters ! Thanks for filing an issue! Unfortunately, I could not reproduce your error following the steps you described above. I tried both with the version you mention (`8.10.1`) and with the latest version (`10.0.2`).
Can you please try to reproduce using the latest version of [`create-nx-workspace`](https://www.npmjs.com/package/create-nx-workspace) (by running `npx create-nx-workspace@latest`)?

If the problem persists, can you please verify that you have the latest version of [`npx`](https://www.npmjs.com/package/npx) as well?

Please try these steps and let me know if they were helpful!

**Comment by bartduisters** [comment]
I am on npx 6.14.5, but that does not seem to be the problem.
With npx 6.14.5 I did the following steps in a terminal:

- npx create-nx-workspace@latest (chose for the angular template)
- npm install --dev @nrwl/storybook
- npx nx g @nrwl/angular:library test-lib
- npx nx g @nrwl/angular:storybook-configuration test-lib
- npx nx run test-lib:storybook

And I was able to start that without any issue.

So, I guess in newly generated workspaces it is not a problem. If anyone experiences the same issue, they can use the workaround provided.
